src/tests/CommonTestRunner/CommonTestRunner.csproj

```diff
@@ -45,6 +41,10 @@
     <Message Importance="High" Text="Created config file $(CommonTestRunnerConfigFileName)" />
     <ItemGroup>
       <FileWrites Include="$(CommonTestRunnerConfigFileName)" />
+      <Content Include="$(CommonTestRunnerConfigFileName)">
+        <Link>Debugger.Tests.Common.txt</Link>
+        <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      </Content>
```

Comment:
```suggestion
```

It seems `Content` doesn't need.

Reply:
This still doesn't ensure the file gets copied. This essentially makes it such that the file gets copied if this target runs before the copy target. This is essentially what the old code did except it won't break if the file is not there. The break is intentional.

Reply:
Yes you are right. I'm closing this as the fix is not correct.